xwiki-platform-core/xwiki-platform-localization/xwiki-platform-localization-script/src/main/java/org/xwiki/localization/script/LocalizationScriptService.java

```diff
@@ -267,4 +269,15 @@ public String render(String key, Syntax syntax, Collection<?> parameters, Locale
 
         return result;
     }
+
+    /**
+     * @return the list of all known locales
+     * @since 9.7RC1
+     * @since 8.4.6
+     * @since 9.6.1
+     */
+    public ULocale[] getKnownLocales()
+    {
+        return ULocale.getAvailableLocales();
```

Comment:
@tmortagne told me there is a bridge provided by ICU to use plain old `Locale` instead of `ULocale`.

It miss the `@Unstable` tag too.